overthebox/files/etc/hotplug.d/iface/32-wanip

```diff
@@ -2,7 +2,7 @@
 #
 # Update wanip on interfaces state change
 #
-WANIP=$(curl --connect-timeout 1 ifconfig.ovh 2>/dev/null)
+WANIP=$(curl --max-time 1 ifconfig.ovh 2>/dev/null)
```

Comment:
We can remove this hotplug script as we use overtheboxd to update wan ip every 30s